web/src/components/apps/AppConfig.jsx

```diff
@@ -652,13 +652,21 @@ class AppConfig extends Component {
             })}
           </div>
           <div className="ConfigArea--wrapper">
-            <IsHelmManaged>
-              {({ isHelmManaged }) => {
-                const { saveConfig, isSaving, error } = useSaveConfig({
+            <UseIsHelmManaged>
+              {({ data = {} }) => {
```

Comment:
I removed the bespoke api for this hook and use `react-query`'s api- with this approach it's up to the component using the hook to rename generic variables like `isError` to avoid conflicts. This ensures that anyone using a hook that relies on `react-query` will know what the hook api is without looking at the implementation.